### PR TITLE
Fixed Review of Issue #3: Traversal of the space of train/test splits

### PR DESCRIPTION
**Pull request by Bolaji61**
One consideration for further exploration is how you would investigate the statement that 0.4/0.6 is an anomaly in your conclusions. Perhaps multiple runs with a variable random seed to see how the performance at different split ratios is affected by the random assignment into test vs train portions is worth investigating.

Above was the previous review made by @mlopatka in #101 
I have researched on the above review and in this PR, I have been able to analyze the effect of different random state numbers (in the train_test_split function) on the performance of the model.

### CODE REVIEW

**Comment by mlopatka** [comment]
Thank you for the PR! Our apologies for the (long) delay in getting to review this contribution.
We took a pause from reviewing contributions made as part of the outreachy application process in order to prioritize planning the roadmap of the PRESC project moving forward. 

As a consequence of this delay, some outstanding PRs exhibit a degree of dissociation from the master branch that makes conflict resolution very complicated. We are capturing contributions that meaningfully contribute to the current direction of the project goals in an archival repo to help us systematically integrate the useful parts into the PRESC master branch. In some cases, this content will be merged and then migrated, and in some cases the specific content will be merged into the archival repo via a separate PR.

Once again, a sincere thanks for you interest and contribution to the PRESC project.
